plugins/tfidf.cc

```diff
@@ -37,7 +37,8 @@ serialize(ML::DB::Store_Writer & store,
     std::string name = "tfidf";
     int version = 0;
     store << name << version;
-    store << corpusSize;
+    store << corpusSize; // number of documents in corpus
+    store << dfs.size(); // number of terms in corpus
```

Comment:
Here I change what we store in the model and *should* ideally change the version.  However, since the tf-idf has not been used yet in prod, I did not do so.